test/ui-automation/wdio.shared.conf.js

```diff
@@ -32,8 +32,8 @@ exports.config = {
   // directory is where your package.json resides, so `wdio` will be called from there.
   //
   specs: [
+    "./test/specs/waci-flow.js",
     "./test/specs/credential-interaction-flow.js",
-    // './test/specs/waci-flow.js',
```

Comment:
Should we remove this line?

Reply:
It's moved up - https://github.com/trustbloc/wallet/pull/1139/files#diff-e6a4ba99822db7ba4d8175cdb5aca2c783f697351046ae0c9de2d1348118a224R35